crates/oxc_linter/Cargo.toml

```diff
@@ -20,16 +20,17 @@ workspace = true
 doctest = false
 
 [dependencies]
-oxc_allocator   = { workspace = true }
-oxc_parser      = { workspace = true }
-oxc_span        = { workspace = true }
-oxc_ast         = { workspace = true }
-oxc_diagnostics = { workspace = true }
-oxc_macros      = { workspace = true }
-oxc_semantic    = { workspace = true }
-oxc_syntax      = { workspace = true }
-oxc_codegen     = { workspace = true }
-oxc_resolver    = { workspace = true }
+oxc_allocator   		= { workspace = true }
```

Comment:
This is a nit, not at all important. But something weird going on with indentation here and in the other `Cargo.toml` files.

Reply:
Nice catch! can you also give this a look https://github.com/oxc-project/oxc/issues/3641#issuecomment-2175834261 ?